DEVELOPMENT.md

```diff
@@ -215,6 +215,36 @@ to start a debugger to which attach from your IDE.
    https://localhost:8081/apis/batch/v1/jobs | jq
    ```
 
+## Enabling Telemetry
+
+1. Install or upgrade KubeArchive adding the following option to the Helm command:
+    ```bash
+    --set "integrations.observability.enabled=true"
+    ```
+    This extra flag makes Helm deploy a traces collector and sets the appropiate environment
+    variables for the KubeArchive deployments.
+1. Forward the traces collector port to localhost:
+    ```bash
+    kubectl port-forward -n kubearchive svc/traces-collector 16686:16686 &
+    ```
+1. Open [http://localhost:16686](http://localhost:16686) in your browser.
```

Comment:
What does this do? Is it to login or is there some other purpose to connecting to this URL? I ask because I wonder if this step is just for viewing the traces. If it is just for viewing the traces, this step should be after the step for enabling Telemetry for Knative Eventing.

Reply:
Its just to open the Jaeger UI to search for traces. I can move it, no problems.